src/Kentico.Search/SearchResultItem.cs

```diff
@@ -38,6 +39,16 @@ public string Content
 
 
         /// <summary>
+        /// Gets the image GUID associated with the search item. Contains data from the field configured as 'Image field' in search index configuration.
+        /// </summary>
+        public Guid Image
```

Comment:
I would expect that property Image will serve Image or something of type Image. IMHO it should be named more clearly like ImageGuid.